src/main/scala/fs2/Chunk.scala

```diff
@@ -29,6 +29,16 @@ trait Chunk[+A] { self =>
     iterator.map(f).copyToBuffer(buf)
     Chunk.indexedSeq(buf)
   }
+  /**
+   * `[[scanLeft]]` is a helper for process1.scan; it differs
+   * from standard collection behavior by not including
+   * the initial value in the output.
```

Comment:
If we implement drop in a similar manner to that in class Doubles, I wouldn't be worried, I just don't want to double memory allocations here.